Info-Orbs/src/main.cpp

```diff
@@ -81,6 +82,7 @@ void setup() {
   widgetSet->add(new ClockWidget(*sm));
   widgetSet->add(new StockWidget(*sm));
   widgetSet->add(new WeatherWidget(*sm));
+  widgetSet->add(new WebDataWidget(*sm));
```

Comment:
We are going to need to build a config option to order the widgets or disable unwanted ones soon.